core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -85,16 +85,10 @@ String process(String value) {
             justification = "app is run in same security context as user providing the filename")
         @Override
         String process(String value) {
-          Scanner scanner = null;
-          try {
-            scanner = new Scanner(new File(value), UTF_8);
+          try (Scanner scanner = new Scanner(new File(value), UTF_8)) {
```

Comment:
This is a good opportunity to use `var`:

```suggestion
          try (var scanner = new Scanner(new File(value), UTF_8)) {
```